modules/oe_theme_content_entity_organisation/oe_theme_content_entity_organisation.install

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Module install file.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+
+/**
+ * Implements hook_install().
+ */
+function oe_theme_content_entity_organisation_install() {
+  // If we are installing from config, we bail out.
+  if (\Drupal::isConfigSyncing() === TRUE) {
+    return;
+  }
+
+  // Override entity view displays.
+  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_entity_organisation') . '/config/overrides');
+  $displays = [
+    'core.entity_view_display.oe_organisation.oe_stakeholder.default',
+  ];
+
+  foreach ($displays as $display) {
+    $values = $storage->read($display);
+    $config = EntityViewDisplay::load($values['id']);
+    foreach ($values as $key => $value) {
+      $config->set($key, $value);
+    }
+    $config->save();
```

Comment:
Instead of this let's use the storage itself to update records such as:
`$view_display_values = $storage->read($view_display_id);`
`$view_display = EntityViewDisplay::load($view_display_values['id']);`
`$updated_view_display = $entity_storage->updateFromStorageRecord($view_display, $view_display_values);`
`$updated_view_display->save();`

Reply:
Make sure that the variable names are truly reflect their content, such as the $display is in your code is a $display_id to be more precise.